test/torchaudio_unittest/transforms/torchscript_consistency_impl.py

```diff
@@ -61,6 +61,7 @@ def test_AmplitudeToDB(self):
 
     def test_MelScale(self):
         spec_f = torch.rand((1, 201, 6))
+        self._assert_consistency(T.MelScale(), spec_f)
         self._assert_consistency(T.MelScale(n_stft=201), spec_f)
```

Comment:
The test to be added is for checking the different aspect of the `MelScale` class.
Therefore, we want to add it as a separate test method.

Let's call this new test `def test_MelScale_invalid(self)`, inside of this test, we want to make sure that scripting `MelScale` with empty `fb` would fail. i.e. `torch.jit.script(T.MelScale())`, should fail.

For this we can use [TestCase.assertRaises](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises).